src/main.ts

```diff
@@ -48,22 +51,162 @@ async function main() {
     throw new Error('Please, set issues in PR title');
   }
 
+  const foundIssues = await Promise.all(foundIssuesIds.map((id) => linearIssueFind(id)));
+
+  // just the declarative wrapper above promise array
+  // we collect the promises here, and await them at the end
+  const jobs = createJobs();
```

Comment:
```suggestion
  const jobs = createTaskRunner();
```

ты же создаешь не задачи, а раннер, который их будет выполнять